draft-irtf-cfrg-ristretto.xml

```diff
@@ -55,10 +54,10 @@ subtle issues.  For instance, Ed25519 implementations may have
 different validation behaviour between batched and singleton
 verification, and at least as specified in <xref target="RFC8032"></xref>, the set of
 valid signatures is not defined by the standard.</t>
-<t>For more complex protocols, careful analysis is required for each
-protocol, as the original security proofs may no longer apply, and the
+<t>More complex protocols require careful analysis as the original
+security proofs may no longer apply, and the
```

Comment:
Happy to simplify this sentence but I would prefer to keep "For more complex protocols" to link and juxtapose with the previously-mentioned "for simple protocols".  Maybe

> For more complex protocols, careful analysis is required as the original security proofs may no longer apply, and the tweaks for one protocol may have disastrous effects when applied to another (for instance, the octuple-spend vulnerability in Monero).